src/styles/blocks/calendar.scss

```
@@ -0,0 +1,40 @@
+
+.calendar {
+  position: relative;
+  display: flex;
+  flex-wrap: wrap;
+  gap: $gap;
+  max-width: $calendar-max-width;
+
+  @each $name, $value in $days {
+    &--day_#{$name} > &__day:first-child {
+      margin-left: $value;
+    }
+  }
+
+  @for $var from $month-length-min to $month-length-max {
+    &--month-length_#{$var} > &__day:nth-child( n + #{$var + 1}) {
```

```suggestion
    &--month-length_#{$var} :nth-child( n + #{$var + 1}) {
```
it could be made simpler